cinema/serializers.py

```diff
@@ -1,6 +1,76 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Genre, Actor, CinemaHall
+
+
+class GenreSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    name = serializers.CharField(max_length=255)
+
+    def create(self, validated_data):
+        return Genre.objects.create(**validated_data)
+
+    def update(self, instance, validated_data):
+        instance.name = validated_data.get("name", instance.name)
+        instance.save()
+        return instance
+
+    class Meta:
+        model = Genre
+        fields = ("id", "name")
```

Comment:
The `Meta` class is not applicable for `serializers.Serializer`. Consider using `serializers.ModelSerializer` if you want to specify the model and fields.